Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -100,6 +100,14 @@
       <redirect />
     </navigation-case>
   </navigation-rule>
+  <navigation-rule>
+    <from-view-id>/*</from-view-id>
+    <navigation-case>
+      <from-outcome>exchangeManager</from-outcome>
```

Comment:
Take out the Connectionmanager.xhtml from navigation-rule.

Reply:
connectionManager.xhtml had been removed